sdks/python/apache_beam/yaml/integration_tests.py

```diff
@@ -69,7 +69,7 @@ def temp_bigquery_table(project, prefix='yaml_bq_it_'):
   dataset_id = '%s_%s' % (prefix, uuid.uuid4().hex)
   bigquery_client.get_or_create_dataset(project, dataset_id)
   logging.info("Created dataset %s in project %s", dataset_id, project)
-  yield f'{project}:{dataset_id}.tmp_table'
+  yield f'{project}.{dataset_id}.tmp_table'
```

Comment:
Was this change needed?

Reply:
IIRC this is reused by EnrichmentTransform's BQ handler which uses the non-legacy form and BigQueryIO accepts both, so this should be safe